sound-design.MD

```diff
@@ -0,0 +1,42 @@
+# MRE Sound API proposal
+
+This is node.js side only. It doesn't actually do anything yet.
+
+## Required functionality
+* preload sounds (as looping or one-shot)
+* play preloaded sounds
+* load and play sounds (Basically preload, instantly play, unload upon completion)
+* pause, resume, 
+* stop sounds 
+* set the sound state (to change volume, pitch, and where possible seek)
+
+## Design thoughts
+
+(Feel free to comment on each item separately)
+
+I'm using assetmanager to for the preloading sound to ask the client to load a sound resource, and return a handle. This is therefore are not instant, but return a promise.
+
+All sounds play on an actor, and the play calls therefore happen on the actor.
```

Comment:
I was debating this. I think that until we have a way to give permission for global changes (like directional light, ambient, skybox), we wouldn't support global sounds. In the AltspaceVR case, it would be fine to support this, but this doesn't truly fit one of the main ideas behind MRE, that MREs are holograms of some finite size

---
In reply to: [256634664](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/191#discussion_r256634664) [](ancestors = 256634664)